src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/TitleValidator.cs

```diff
@@ -2,28 +2,29 @@
 using Esfa.Vacancy.Domain.Validation;
 using FluentValidation;
 
-
 namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship.Validators
 {
-    public class TitleValidator : AbstractValidator<string>
+    public partial class CreateApprenticeshipRequestValidator
     {
-        private const string PropertyName = "Title";
         private const string TitleApprentice = "apprentice";
         private const int TitleMaximumLength = 100;
 
-        public TitleValidator()
+        private void TitleValidator()
         {
-            RuleFor(x => x)
-                .MaximumLength(TitleMaximumLength)
-                .WithErrorCode(ErrorCodes.CreateApprenticeship.TitleMaximumFieldLength)
-                .WithName(PropertyName)
+            RuleFor(request => request.Title)
+                .NotEmpty()
+                .WithErrorCode(ErrorCodes.CreateApprenticeship.TitleIsRequired)
+
+                .DependentRules(rules => rules.RuleFor(request => request.Title)
+                    .MaximumLength(TitleMaximumLength)
+                    .WithErrorCode(ErrorCodes.CreateApprenticeship.TitleMaximumFieldLength)
 
-                .Must(title => title.IndexOf(TitleApprentice, StringComparison.OrdinalIgnoreCase) >= 0)
-                .WithErrorCode(ErrorCodes.CreateApprenticeship.TitleShouldIncludeWordApprentice)
-                .WithName(PropertyName)
-                .WithMessage(ErrorMessages.CreateApprenticeship.TitleShouldIncludeWordApprentice)
+                    .Must(title => title.IndexOf(TitleApprentice, StringComparison.OrdinalIgnoreCase) >= 0)
```

Comment:
Can you not use string.Contains?

Reply:
tbh I wasn't looking so much at the logic of each existing rule. I'll have a quick test, of your suggestion.

Reply:
`.Must(title => title.ToLowerInvariant().Contains(TitleApprentice.ToLowerInvariant()))`
not sure if that reads any better.

Reply:
so string.Contains doesn't have the (string, comparer) overload but does for (char, comparer) 😞

Reply:
Leave your code as is then